rest/zone_test.go

```diff
@@ -9,80 +11,148 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestPagination_zoneList(t *testing.T) {
+func TestZone_list_pagination(t *testing.T) {
```

Comment:
Snake case function names is weird.  Should probably be named `TestZone_ListPagination`